websites/tasks.py

```diff
@@ -0,0 +1,68 @@
+""" Tasks for websites """
+import logging
+
+import boto3
+import celery
+from django.conf import settings
+
+from main.celery import app
+from main.utils import chunks
+from websites.api import import_ocw2hugo_course
+
+log = logging.getLogger(__name__)
+
+
+@app.task(bind=True)
+def import_ocw2hugo_course_paths(
+    self, paths=None, bucket=None, prefix=None
+):  # pylint:disable=unused-argument
+    """
+    Import all ocw2hugo courses & content
+
+    Args:
+        paths (list): list of course url paths
+        bucket (str): S3 bucket name
+        prefix (str): S3 prefix before start of course_id path
+
+    """
+    if not paths:
+        return
+    for path in paths:
+        log.debug("Importing %s", path)
+        import_ocw2hugo_course(bucket, prefix, path)
+
+
+@app.task(bind=True)
+def import_ocw2hugo_courses(self, bucket=None, prefix=None, chunk_size=100):
+    """
+    Import all ocw2hugo courses & content
+
+    Args:
+        bucket (str): S3 bucket name
+        prefix (str): S3 prefix before start of course_id path
+        chunk_size (int): Number of courses to process per task
+    """
+    if not bucket:
+        raise TypeError("Bucket name must be specified")
+    s3 = boto3.resource(
+        "s3",
+        aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+    )
```

Comment:
Seeing similar code to this in a few places, it'd probably be good to pull this out into a utility method. [`boto3.session.Session`](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html#boto3.session.Session) would probably be useful here too.